tests/function/test_service.py

```diff
@@ -91,3 +95,70 @@ def test_force_reload(self, virtwho):
         _, _ = virtwho.operate_service(action='force-reload')
```

Comment:
actually, we can just use 
`virtwho.operate_service(action='force-reload')` here

Reply:
but it is fine we can improve this in the future